tasks/cea-install.js

```diff
@@ -48,35 +51,82 @@ function install() {
   });
 }
 
+// Create new package.json
+const npmInit = async () => {
+  const templatePkg = require(path.join(__dirname, '..', 'package.json'));
+  const appPkg = {
+    name: TARGET_DIR,
+    version: '0.1.0',
+    private: true
+  };
+
+  appPkg.scripts = templatePkg.scripts;
+  appPkg.devDependencies = templatePkg.devDependencies;
+  appPkg.dependencies = templatePkg.dependencies;
+  appPkg.eslintConfig = templatePkg.eslintConfig;
+  await fs.writeFileSync(
+    path.join(TARGET_DIR, 'package.json'),
+    JSON.stringify(appPkg, null, 2) + os.EOL
+  );
+};
+
+// Copy template files to target
+const srcInit = async () => {
+  const copyDirs = [
+    'src',
+    '.editorconfig',
+    '.eslintrc',
+    '.gitignore',
+    'babel.config.js',
+    'karma-test-shim.js',
+    'karma.conf.js',
+    'lws.config.js',
+    'postcss.config.js',
+    'README.md',
+    'webpack.config.common.js',
+    'webpack.config.develop.js',
+    'webpack.config.prod.js'
+  ];
+
+  return await Promise.all(
+    copyDirs.map(async directory => {
+      const templateDir = path.join(__dirname, '..', directory);
+
+      if (await fs.existsSync(templateDir)) {
+        return await fs.copySync(
+          templateDir,
+          path.join(TARGET_DIR, directory)
+        );
+      } else {
+        console.error("Directory doesn't exist! :" + templateDir);
+        process.exit(); // eslint-disable-line no-process-exit
```

Comment:
curious why we wouldn't want to [pass `1`](https://stackoverflow.com/a/5266239/417806) in all these instances of `process.exit`?